Assets/MixedRealityToolkit.SDK/Features/Utilities/PointerUtils.cs

```diff
@@ -144,6 +144,159 @@ public static IEnumerable<IMixedRealityPointer> GetPointers()
                 }
             }
         }
-    }
 
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the head gaze.
+        /// </summary>
+        /// <param name="ray">The physics ray for the pointer</param>
+        /// <returns>
+        /// True if the ray contains valid data, false otherwise.
+        /// </returns>
+        public static bool TryGetHeadGazeRay(out Ray ray)
+        {
+            return TryGetRay(InputSourceType.Head, Handedness.None, out ray);
+        }
+
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the eye gaze.
+        /// </summary>
+        /// <param name="ray">The physics ray for the pointer</param>
+        /// <returns>
+        /// True if the ray contains valid data, false otherwise.
+        /// </returns>
+        public static bool TryGetEyeGazeRay(out Ray ray)
+        {
+            return TryGetRay(InputSourceType.Eyes, Handedness.None, out ray);
+        }
+
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the user's gaze
```

Comment:
This is interesting, but I'm actually a bit confused by this because why is it that only one input is active? For example, if I'm on HoloLens 2 and have my eyes calibrated, then both my eyes and my head should be available. So, what would the method return if I'm on HoloLens 2 and have eyes calibrated?

I can see the desire here might be for devs to understand "is the system using the eye pointer or the gaze pointer as the main cursor", but I think that might be convolving the purpose of the "get eye ray" and "get head gaze ray" methods, which are meant to just return where the user's head / eyes are facing, regardless of what input pointers are active.

Reply:
fixed.